Part2_Mapplots/ex2_2_psyplot.ipynb

```diff
@@ -193,9 +193,10 @@
     "ax = plt.axes(projection=ccrs.Robinson())\n",
```

Comment:
<p>Mention a few words (in the text) about the cmcrameri library before we use it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Added a short explanation and adapted the exercise slightly, so that the users can choose their favorite colormap.

Reply:
What is the cmcrameri library?

Reply:
<p>I added the exercise:</p><p><code>Let's use a nicer colormap. The Python wrapper [cmcrameri](https://github.com/callumrollo/cmcrameri) was developed to use [scientific colour maps](https://www.fabiocrameri.ch/colourmaps/) in Python. Have a look at the available colour maps and choose one you like.</code></p>

Reply:
Great!